qpc/messages.py

```diff
@@ -37,14 +36,21 @@
 CRED_VC_PWD_AND_USERNAME = 'VCenter requires both username and password.'
 CRED_VC_EDIT_PWD_OR_USERNAME = 'Must update either username or password '
 'for VCenter credentials'
-CRED_VC_KEY_FILE_NOT_ALLOWED = 'VCenter cannot use a sudo password, ssh'\
+CRED_VC_KEY_FILE_NOT_ALLOWED = 'VCenter cannot use a become password, ssh'\
```

Comment:
We also need to include become_method and become_user.

I would think Satellite should have a similar message too.

Reply:
I couldn't find a similar message for satellite in this file. But I am also unsure of where this message is getting used because I changed the name to 'CRED_VC_FIELDS_NOT_ALLOWED and I didn't get any lint errors & it didn't make any of the tests fail -- so maybe it isn't getting used?